test/unit/pending_blocks_test.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Unit.PendingBlocks do
   @moduledoc false
 
-  use ExUnit.Case, async: true
```

Comment:
we shouldn't do async: true when using Patch because it will complain of more than one Patch supervisor running